xml/admin_security.xml

```diff
@@ -783,6 +783,223 @@ roleRef:
    </example>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="sec.admin.security.pod_policies">
+  <title>Pod Security Policies</title>
+  <para>
+   This section provides an overview of policy settings for pod
+   security. By default, pod security policies are already enabled on
+   &productname;.
+  </para>
+  <para>
+   &productname; comes with 2 defined policies. The required role
+   definition is also included in the examples. Role bindings have
+   to be created manually. For details about roles and role bindings,
+   refer to <xref linkend= "sec.admin.security.role" />.
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term><xref linkend="ex.admin.security.pod_policies.definition.unprivileged"/></term>
+    <listitem>
+     <para>
+      This is the default policy. It is a compromise between security
+      and daily needs. This policy is bound to the role
+      <literal>suse:caasp:psp:unprivileged</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><xref linkend="ex.admin.security.pod_policies.definition.privileged"/></term>
+    <listitem>
+     <para>
+      This policy has few restrictions and should only be given to
+      highly trusted users. This policy is bound to the role
+      <literal>suse:caasp:psp:privileged</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   To adapt the provided example policies to your needs, copy them
```

Comment:
It's not possible to change the included PSPs (and have those changes persist). If the customer needs to tweak the policies, they must create a new PSP (optionally, using one of ours as a starting point).